tests/e2e.rs

```diff
@@ -33,7 +35,11 @@ pub fn assert_is_same<A: AsRef<Path>, B: AsRef<Path>>(
     expected_base: B,
     output: &std::process::Output,
 ) -> Result<(), Box<dyn Error>> {
-    let diffs = dir_diff_list::search_diff(actual_base, expected_base, &[])?;
+    let diffs = dir_diff_list::search_diff(
+        actual_base,
+        expected_base,
+        &[PathPattern::from_str(".ffizer.d")?],
```

Comment:
Why ignore this folder? Instead of including it in the expected. It's a good way to test it. Or maybe just ignore the `version.txt` to avoid updating it for every version.

Otherwise, I agree that the sub-command `test-sample` should ignored.